src/domains/ecommerce-app/04_apim_ecommerce.tf

```diff
@@ -276,6 +276,25 @@ locals {
   }
 }
 
+
+module "apim_ecommerce_payment_methods_product" {
```

Comment:
can you move the product definition to the beginning of the file where the other product is also defined? more or less 32 line

Reply:
Solved by https://github.com/pagopa/pagopa-infra/pull/1973/commits/f79776bf4b36720e5bcaf93515c3964909eff890